src/frontend/Komponenter/Personoversikt/HistorikkVedtaksperioder/VedtaksperioderBarnetilsyn.tsx

```diff
@@ -37,7 +48,7 @@ const historikkRad = (andel: AndelHistorikk) => {
             <td>{andel.saksbehandler}</td>
             <td>
                 <Link className="lenke" to={{ pathname: `/behandling/${andel.behandlingId}` }}>
-                    {erMigrering ? 'Migrering' : behandlingstypeTilTekst[andel.behandlingType]}
+                    {behandlingstypeTilTekst[andel.behandlingType]}
```

Comment:
Har du lagt inn migrering i behandlingstypeTilTekst? 🤔

Reply:
Hvis ikke vil det ikke stå noe på de som har "Migrering" i dag?

Reply:
Vi har ikke migrering for Barnetilsyn, men den finnes for Overgangsstønad

Reply:
Ja måtte ha det pga kolonnene :P